pubtools/_pulp/tasks/set_maintenance/set_maintenance_off.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import re
+
+from .base import SetMaintenance
+
+
+LOG = logging.getLogger("set-maintenance")
+
+
+class SetMaintenanceOff(SetMaintenance):
+    """Unset repositories maintenance mode.
+
+    See "pub maintenance-on --help" for more information on maintenance mode.
+    """
+
+    def run(self):
+        report = self.pulp_client.get_maintenance_report().result()
+
+        to_remove = []
+        if self.args.repo_regex:
+            for entry in report.entries:
+                if re.match(self.args.repo_regex, entry.repo_id):
```

Comment:
yes, ran regex against the URL sounds more useful. 
The reason I use regex against id is because there is no URL in the maintenance report, every entry only contains id, message and a timestamp. For removing entries from the report, run regex against URL is not usable and then the user needs to pass all repo ids they want to remove from maintenance mode. 

I'm thinking I need to add URL to maintenance entry?

Reply:
I don't think you need to add URLs to the maintenance entries - the entry gives you the repo ID and that gives you enough info to find the repos, which then allows you to check their URLs.  Or do you have a concern with doing it that way?  It doesn't seem like there'd be a performance issue, it only costs one additional Pulp repo search.

Reply:
ah, makes sense.